internal/cli/apis.go

```diff
@@ -249,7 +280,16 @@ auth0 apis update <id|audience> --name myapi`,
 				return err
 			}
 
-			api := &management.ResourceServer{}
+			if err := apiTokenLifetime.Ask(cmd, &inputs.TokenLifetime, nil); err != nil {
```

Comment:
```suggestion
			if err := apiTokenLifetime.AskU(cmd, &inputs.TokenLifetime, nil); err != nil {
```
Use `AskU` for update command.